build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -219,14 +219,21 @@ public void ReleaseLeaseOnBlob(string blob, string leaseId)
             cloudBlob.ReleaseLeaseAsync(ac).Wait();
         }
 
-        public bool IsLatestSpecifiedVersion(string version)
+        public bool DoesBlobReferenceExist(string version)
```

Comment:
There is nothing version specific in the implementation of the function - it simply checks for  a string reference in the blob.